docs/admin/react-hooks.mdx

```diff
@@ -711,23 +711,41 @@ const CustomComponent: React.FC = () => {
-| **`getDocPermissions`**   | Method to retrieve document-level user preferences.                                                 |
-| **`getDocPreferences`**   | Method to retrieve document-level user preferences.                                                 |
-| **`hasPublishedDoc`**     | Whether the document has a published version.                                                                      |
-| **`incrementVersionCount`** | Method to increment the version count of the document.                                                           |
-| **`preferencesKey`**      | The `preferences` key to use when interacting with document-level user preferences.                                 |
-| **`versions`**            | Versions of the current doc.                                                                                        |
-| **`unpublishedVersions`** | Unpublished versions of the current doc.                                                                            |
-| **`publishedDoc`**        | The currently published version of the doc being edited.                                                            |
-| **`getVersions`**         | Method to retrieve document versions.                                                               |
-| **`docPermissions`**      | The current documents permissions. Collection document permissions fallback when no id is present (i.e. on create). |
-| **`versionCount`**        | The current version count of the document.                                                                          |
+| Property                            | Description                                                                                             |
+| ----------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| **`action`**                        | The current action being performed on the document (create, edit, etc.).                                |
+| **`apiURL`**                        | The API URL for the current document.                                                                   |
+| **`collectionSlug`**                | The slug of the collection if editing a collection document.                                            |
+| **`currentEditor`**                 | The user currently editing the document.                                                                |
+| **`docConfig`**                     | Either the Collection or Global config of the document, depending on what is being edited.              |
+| **`docPermissions`**                | The current document's permissions. Fallback to collection permissions when no id is present.           |
+| **`documentIsLocked`**              | Whether the document is currently locked by another user.                                               |
```

Comment:
Let's add a link to the Locked Documents documentation here.

```suggestion
| **`documentIsLocked`**              | Whether the document is currently locked by another user. [More details](./locked-documents).                                             |
```

This same change should be applied to all items related to the Locked Documents feature, e.g. `unlockDocument`, `updateDocumentEditor`, etc.